src/common/components/body-class-modifier.tsx

```diff
@@ -17,7 +17,7 @@ export type BodyClassModifierProps = {
 // body classes office fabric maintains.
 export class BodyClassModifier extends React.Component<BodyClassModifierProps> {
     public render(): JSX.Element {
-        return null;
+        return <></>;
```

Comment:
If there is a constant somewhere for returning an empty React node/fragment, please let me know.

Reply:
`null` is canonical; I think the better fix here would actually be to expand the return type domain (if you look at the `React.Component` typings, `render` is actually allowed to return a `ReactNode`, which `null` is a valid option for; the compiler allows us to use `JSX.Element` without complaint because it is a subtype of `ReactNode`, but so is `JSX.Element | null`)